css/style.css

```diff
@@ -436,3 +446,247 @@ header {
+    font-size: 15px;
+    line-height: 20px;
+    width: auto;
+    height: 20px;
+    color: #344563;
+    transition: all 1s;
+  }
+
+  .headline {
+    background: #fff url(../images/header-bg-desktop.svg) no-repeat 100% 20%;
+    width: 100%;
+    height: 57.25em;
+    align-items: flex-start;
+    padding: 290px 260px 289px 260px;
+  }
+
+  .headline-title {
+    font-size: 48px;
+  }
+
+  .card {
+    justify-items: center;
+    margin: 142px;
+    gap: 142px;
+  }
+
+  .card-section {
+    width: fit-content;
+    height: max-content;
+    flex-direction: row;
+    justify-content: center;
+    gap: 38px;
+    padding-top: 24px;
+    border: 3px solid #dfe1e6;
+    border-radius: 16px;
+    box-shadow: 0 0 0.5px 0.25px inset #c8c6c6;
+  }
+
+  .work-section-texts {
+    height: auto;
+    padding: 24px;
+    gap: 24px;
+  }
+
+  #reverce {
+    flex-direction: row-reverse;
+  }
+
+  .heading-title {
+    width: 496px;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .desciption-p-mobile {
+    width: 450px;
+    height: auto;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .btn-see-project {
+    height: auto;
+    line-height: 36px;
+    width: max-content;
+  }
+
+  .btn-see-project:hover {
+    border: 1px solid #ebebff;
+    background: #6070ff;
+    color: white;
+  }
+
+  .card-sub-container {
+    gap: 25px;
+  }
+
+  .title-canopy {
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 24px;
+  }
+
+  .title-developer {
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .title-year {
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .about-section {
+    display: grid;
+    grid-template-columns: 40% 60%;
+    grid-template-rows: auto;
+    padding: 142px;
+    align-items: baseline;
+  }
+
+  .contact-container {
+    width: 100%;
+    height: 52.875em;
+    background: #6070ff url(../images/contact-bg-form-desktop.svg) no-repeat 65% 65%;
+  }
+
+  .contact-container-sub {
+    padding-top: 70px;
+  }
+
+  .lange-type {
+    flex-direction: row;
+  }
+
+  .lange-type-logo {
+    width: max-content;
+    height: 120px;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .lang-list {
+    width: 100%;
+  }
+
+  .lang-list-frame {
+    width: 100%;
+  }
+
+  .contact-info {
+    height: auto;
+    width: 569px;
+    margin-bottom: 12px;
+  }
+
+  .contact-form input {
+    width: 448px;
+  }
+
+  #message-box {
+    width: 448px;
+  }
+
+  .contact-form button {
+    align-self: center;
+    padding: 9px;
+  }
+
+  .desktop-menu-item {
+    font-family: "Poppins", sans-serif;
+    text-decoration: none;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    width: auto;
+    height: 20px;
+    color: #344563;
+    transition: all 1s;
+  }
+}
+
+@media screen and (max-width: 767px) {
```

Comment:
- [ Required ] As noticed by the previous reviewer I have also noticed that on-screen sizes less than `767px` your design introduces a horizontal scroll on the webpage. This is because the desktop menu is not hidden on mobile devices, if you were to hide the desktop menu on smaller devices the horizontal scroll would disappear. You can fix this by applying the CSS rule below in your desktop media query.

```CSS
@media screen and (max-width: 767px) {
  .desktop-menu {
    display: none;
     }
  }
```

## Horizontal scroll on mobile devices
https://user-images.githubusercontent.com/16392046/213808503-265167e8-39e2-441d-9296-faccf84d183d.mp4